tools/Makefile.unix

```diff
@@ -324,6 +324,14 @@ tools/cnvwindeps$(HOSTEXEEXT):
 # 	$(Q) echo "No .config file found, creating one"
 # 	$(Q) tools/initialconfig$(HOSTEXEEXT)
 
+# make it easy to see what is happening
+COLOR_BLUE = \033[0;94m
```

Comment:
There is no coding standard governing Makefiles.  Other locations in this file do put spaces after comments, just like the C requirement.

Reply:
Done. Yes there seems to be the "what you do" and what we have documented - disconnect.
# Please use the "Suggested Change Button or ctr-g"

Reply:
@davids5 I would prefer we work on check_code_style.sh first, and forget Makefile.unix at all.
Why? this avoid we couple the script logic with Makefle, for example:
1.The above implementation make the standalone script can't colorize the output.
2.The dependence rule can't work if user invoke the script from shell directly.
The best aproach is that we focus on the script development and make it work from command line as we want, then we can integrate the script either as the make target or precommit hook. Otherwise, the script will do a major rework for the standalone case.
Invoking the script from command line is the key feature for this tool, becasue we need integrate this tool the various workflow(e.g. Makefile, precommit hook, jenkins job).

Reply:
That actually makes a lot of sense, is simpler, and sounds more flexible.  I would tend to support Xiao Xiangs argument.

Reply:
Unresolved only because I think we need to address Xiao Xiang's recommendation in some fashion.